crates/cxx-qt-gen/src/generator/cpp/property/mod.rs

```diff
@@ -186,15 +177,14 @@ pub mod tests {
         )
         .unwrap();
 
-        // should be a pair
-        let result = require_header(&generated.methods[0]);
-        assert!(result.is_err());
-
-        let result = require_pair(&generated.private_methods[0]);
+        // should be a header
```

Comment:
this comment is now incorrect?

Reply:
No, it's correct. We're asserting that `require_pair` returns an error, because it's actually a header.

I think this just exists to cover all branches of the `require_pair` function...

Reply:
Ah it's hitting the errors states i see.